docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -263,56 +263,36 @@ For more information, see [Installing Grafana Mimir dashboards and alerts].
 
 The `mimir-distributed` Helm chart makes it easy for you to collect metrics and
 logs from Mimir. It assigns the correct labels for you so that the dashboards
-and alerts simply work. The chart uses the Grafana Agent to ship metrics to
-a Prometheus-compatible server and logs to a Loki or GEL (Grafana Enterprise
-Metrics) server.
+and alerts work.
 
-1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from
-   https://github.com/grafana/agent/tree/main/operations/agent-static-operator/crds
-2. Install the CRDs on your cluster:
+{{< admonition type="caution" >}}
+The `mimir-distributed` Helm chart uses Grafana Agent, which is now deprecated. Use Grafana Alloy to ship metrics and logs instead.{{< /admonition >}}
 
-   ```bash
-   kubectl apply -f operations/agent-static-operator/crds/
-   ```
-
-3. Add the following YAML snippet to your
-   values file, to send metamonitoring telemetry from Mimir. Change the URLs and credentials to match your desired
-   destination.
+To send metamonitoring telemetry from Mimir, add the following YAML snippet to your
+   values file:
 
    ```yaml
    metaMonitoring:
      serviceMonitor:
        enabled: true
      grafanaAgent:
-       enabled: true
-       installOperator: true
-
-       logs:
-         remote:
-           url: "https://example.com/loki/api/v1/push"
-           auth:
-             username: 12345
-
-       metrics:
-         remote:
-           url: "https://prometehus.prometheus.svc.cluster.local./api/v1/push"
-           headers:
-             X-Scope-OrgID: metamonitoring
+       enabled: false
```

Comment:
This `enabled: false` doesn't seem right. It will disable the meta monitoring agent, if I read the [chart's values][1] correctly.

[1]: https://github.com/grafana/mimir/blob/fefc1d5071877ca88390b28852ab4223cf0bc58c/operations/helm/charts/mimir-distributed/values.yaml#L3629-L3634

Reply:
Yes, @narqo we actually want to disable it because it deploys Grafana Agent in Operator mode which are both deprecated. Just keeping the serviceMonitor would allow Mimir to be scraped by a separately deployed Alloy/Prometheus Operator.

If someone wants to modify the chart to deploy Alloy, that could work too but I'm not sure that's the best course since it could be in a shared cluster with Alloy already.

Reply:
There is an issue open to modify the chart to deploy Alloy. See https://github.com/grafana/mimir/issues/5860 and https://raintank-corp.slack.com/archives/C029912SXT8/p1720198051487749. Since these changes are upcoming, I'm going to add a simple deprecation notice to these docs now and then we can update again, as required, when the new chart is ready.